chef/roles/logging-server.rb

```diff
@@ -2,7 +2,7 @@
 name "logging-server"
 description "Logging Servier Role - Logging master for the cloud"
 run_list(
-         "recipe[logging::server]"
+         "recipe[logging::role_logging_server]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.